fixit/fix_it.py

```diff
@@ -82,17 +78,28 @@ def play(self):
                     try:
                         person = int(input("What is your id?\n"))
                         price = int(input("What is the price of the share?\n"))
-                    except:
+                    except (SyntaxError, ValueError):
                         print("Try Again.")
                         continue
                     else:
-                        order = Order(person, price, 1)
                         if buyer == 1:
-                            self.add_order(person, order, "BUY")
+                            self.add_order(person, price, "BUY")
```

Comment:
why does this change?

Reply:
I changed the add_order and delete_order functions to be called with a number for the second parameter rather than an order object so that that it was easier to call these functions in server.py